modules/custom/localgov_restricted_content/src/RestrictedContent.php

```diff
@@ -35,6 +35,12 @@ public function isAncestorRestricted(ContentEntityInterface $entity): bool {
           if ($parent->{$this::RESTRICTED_CONTENT_FIELD}->value) {
             return TRUE;
           }
+          if ($entity->id() == $parent->id()) {
```

Comment:
Ah! Should be `===`.

Lovely bit of code otherwise. :-)

Reply:
Fixed